projects/ngrx.io/content/guide/store/recipes/angular-elements.md

```diff
@@ -0,0 +1,113 @@
+# Using Store with Angular Elements
+
+## Creating Angular Elements
+
+The following recipe illustrates utilizing Angular Elements with a NgRx Store. This provides the ability to manage the state of a counter, and to select and display that counter state from within separate Angular Elements. This is all accomplished from within a single Angular Project and static HTML page.
```

Comment:
```suggestion
The following recipe illustrates utilizing [Angular Elements](https://angular.io/guide/elements) with a NgRx Store. This provides the ability to manage the state of a counter, and to select and display that counter state from within separate Angular Elements. This is all accomplished from within a single Angular Project and static HTML page.
```

Reply:
In the actual example, we wanted to have a sepearet element for every action. Agreed with empty css files let me remove.